javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,39 @@
+const width = 7
+const height = 4
+const area = width * height
+console.log('value of area: ', area)
+console.log('typeof area: ', typeof area)
+
+const bill = 10
+const payment = 4.5
+const change = bill - payment
```

Comment:
Your `payment` should be enough to cover the `bill`. You'll also want to swap line 9, so it's `payment - bill`